components/suggest/src/schema.rs

```diff
@@ -6,7 +6,7 @@
 use rusqlite::{Connection, Transaction};
 use sql_support::open_database::{self, ConnectionInitializer};
 
-pub const VERSION: u32 = 6;
+pub const VERSION: u32 = 7;
```

Comment:
Nice! Could you please bump the range in `upgrade_from`, too, so that we'll throw away version 6 databases? 😄

Reply:
Updated!